client/src/components/survey/SurveyPage1.jsx

```diff
@@ -101,15 +101,15 @@ const marks = [
 ];
 
 const SurveyPage1 = props => {
-  const { setSurveyPage1 } = props;
+  const { setSurveyPage1, dailyfeeling, dailySymptomsFeeling, dailyComparedToYesterday } = props;
   const classes = useStyles();
   const [open, setOpen] = useState(false);
-  const [todaySet, setTodaySet] = useState(false);
-  const [symptomsSet, setSymptomsSet] = useState(false);
-  const [comparedSet, setComparedSet] = useState(false);
-  const [todayFeeling, setTodayFeeling] = useState(-1);
-  const [todaySymptoms, setTodaySymptoms] = useState(-1);
-  const [comparedFeeling, setcomparedFeeling] = useState('');
+  const [todaySet, setTodaySet] = useState(!!dailyfeeling);
+  const [symptomsSet, setSymptomsSet] = useState(!!dailySymptomsFeeling);
+  const [comparedSet, setComparedSet] = useState(!!dailyComparedToYesterday);
```

Comment:
Couldn't we just use `dailyfeeling` here instead of effectively saying `not not dailyfeeling`?

Reply:
Yes! This was based on an earlier way I was doing it, and that's both cleaner and makes more sense.